__tests__/src/helpers/fs.helper.test.ts

```diff
@@ -1,7 +1,7 @@
 import {beforeEach, describe, expect, it, jest} from "@jest/globals";
-import fs from "fs";
-import {fsHelper} from "../fs.helper";
-import {promptObjectInterface} from "../prompt.helper";
+import * as fs from "fs";
```

Comment:
You don't need that * as stuff

Reply:
is is required when import as a namespace